JS/Lesson15/goksel/script.js

```diff
@@ -103,62 +104,66 @@ const stringExercise14 =
 // console.log('Exercise14: ', stringExercise14.slice(
 //   stringExercise14.indexOf('searchString'), -1)
 // );
-
+console.log("Exercises14: " ,stringExercise14.substring(0, stringExercise14.indexOf("are") - 1) );
```

Comment:
There could be another sentence where the word Databases would not be in the beginning of it. Try to also use indexOf to find the start index :)

Reply:
console.log("Exercises14: " ,stringExercise14.substring(stringExercise14.indexOf("Databases"), stringExercise14.indexOf("are") - 1) );